maven-release-plugin/src/main/java/org/apache/maven/plugins/release/PrepareReleaseMojo.java

```diff
@@ -455,7 +453,8 @@ private String getLineSeparatorFromPom()
     {
         char current;
         String lineSeparator = "";
-        try ( InputStream is = new FileInputStream( this.project.getFile() ) )
+        File releasePom = this.project.getFile();
+        try ( InputStream is = new FileInputStream( releasePom ) )
```

Comment:
Why this change if you don't check for the existance? I don't see a benefit...

Reply:
this was a relic from one of the previous commits. I squashed them now and returned this part to its original state.